src/index.js

```diff
@@ -116,6 +121,21 @@ class CreditCardInput extends Component {
     }
   }
 
+  validateExpirationDate (date) {
```

Comment:
Looks like the expiration date fix snuck in somehow!

Reply:
NOooo